packages/story-editor/src/app/media/utils/test/useProcessMedia.js

```diff
@@ -110,6 +110,7 @@ function setup() {
   const storyContextValue = {
     actions: { updateElementsByResourceId },
   };
+
```

Comment:
?

Reply:
Leftover from some previous work on this PR --- cleaned up now.